translations/en.json

```diff
@@ -232,23 +262,17 @@
         "stocksource": "",
         "stocksourcedestination": ""
       }
-    },
-    "yes": "Yes"
+    }
   },
   "stockManagement": "Stock Management",
-  "stockManagementLocations": "Locations",
```

Comment:
does this mean the sidenav menu items will not be translated ?

Reply:
Yes, it is now translated